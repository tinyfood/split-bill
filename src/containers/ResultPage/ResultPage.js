import React, { Component } from 'react'
import PersonsList from '../../components/PersonsList/PersonsList'

export default class ResultPage extends Component {
  render() {
    return (
      <div className="result-page">
        <PersonsList />
      </div>
    )
  }
}
