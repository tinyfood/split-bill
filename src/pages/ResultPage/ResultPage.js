import React, { Component } from 'react'
import PersonsList from '../../components/PersonsList/PersonsList'
import styled from 'styled-components'

const ResultPageWrap = styled.div`
  width: 96%;
  height: 315px;
  margin: 0 auto;
`

export default class ResultPage extends Component {
  render() {
    return (
      <ResultPageWrap>
        <PersonsList />
      </ResultPageWrap>
    )
  }
}
