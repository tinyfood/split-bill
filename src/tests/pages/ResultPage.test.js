import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import ResultPage from '../../pages/ResultPage/ResultPage'

test('should render ResultPage correctly', () => {
  const wrapper = shallow(<ResultPage />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
