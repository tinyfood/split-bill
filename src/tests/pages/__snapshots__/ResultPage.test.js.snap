// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ResultPage correctly 1`] = `
<styled.div>
  <ConnectFunction />
</styled.div>
`;
